dbos/dbos_config.py

```diff
@@ -70,9 +71,22 @@ def replace_func(match: re.Match[str]) -> str:
     return re.sub(regex, replace_func, content)
 
 
-def load_config(configFilePath: str = "dbos-config.yaml") -> ConfigFile:
+def get_dbos_database_url(config_file_path: str = "dbos-config.yaml") -> str:
```

Comment:
Does this file path work? Do we have to use relative path?
```
dbos_config_path = os.path.join(
    os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "dbos-config.yaml"
)
```

Reply:
As we discussed, it's not a huge issue. We need to run the dbos migrate command in the root folder for now.

Reply:
tested w/ Qian watching over shoulder. If the explicit path is not provided, the current directory is used. So if you call `dbos migrate` from the root of project, the config file resolves correctly. 

For now, we're going to leave it like this. Will consider changing based on user feedback